lib/improver/utilities/ancillary_creation.py

```diff
@@ -28,12 +28,192 @@
 # CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE)
 # ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE
 # POSSIBILITY OF SUCH DAMAGE.
-"""Plugins to create ancillary data."""
 
 import iris
 import numpy as np
-from improver.psychrometric_calculations.psychrometric_calculations import (
-    Utilities)
+from improver.psychrometric_calculations.psychrometric_calculations import \
+    Utilities
+from improver.utilities.spatial import DifferenceBetweenAdjacentGridSquares
+
+
+class OrographicAlphas(object):
+
+    """
+    Class to generate alpha smoothing parameters for recursive filtering
+    based on orography gradients.
+    """
+
+    def __init__(self, min_alpha=0., max_alpha=1., coefficient=1, power=1,
+                 invert_alphas=True):
+        """
+        Initialise class.
+
+        Args:
+            min_alpha : float
```

Comment:
All of the args in this code are formatted incorrectly for sphinx. See other code for examples, but we need:
```min_alpha (float):```
```cube (iris.cube.Cube):``` etc.